src/video/windows/SDL_windowswindow.h

```diff
@@ -27,6 +27,14 @@
 #include "../SDL_egl_c.h"
 #endif
 
+typedef enum
+{
+    SDL_MOUSE_EVENT_SOURCE_UNKNOWN,
+    SDL_MOUSE_EVENT_SOURCE_MOUSE,
+    SDL_MOUSE_EVENT_SOURCE_TOUCH,
+    SDL_MOUSE_EVENT_SOURCE_PEN,
+} SDL_MOUSE_EVENT_SOURCE;
+
```

Comment:
I'm not entirely sure if moving the enum here is the right thing to do, given how it's effectively only returned by `GetMouseMessageSource` in `SDL_windowsevents.c`. Also, is it safe to assume that driver data layout shouldn't be relied on by applications?